src/main/java/ChatBot.java

```diff
@@ -0,0 +1,67 @@
+public class ChatBot {
+    private final String BOT_NAME = "Big Bob";
+    private final int MAXIMUM_NUMBER_OF_TASK = 100;
+    private Task[] listOfTasks = new Task[MAXIMUM_NUMBER_OF_TASK];
+    private int numOfTaskInList = 0;
+
+    public ChatBot() {
+        System.out.println("\t Greetings Human! I'm " + BOT_NAME + ".");
+        System.out.println("\t How may i be of service to you?");
+    }
+
+    public void executeCommand(Command inputCommand) {
+        if (inputCommand.getType() == Command.CommandType.ADDTASK) {
+            AddTaskCommand newAddCommand = (AddTaskCommand) inputCommand;
+            addTaskToList(newAddCommand);
+
+        } else if (inputCommand.getType() == Command.CommandType.UPDATETASKSTATUS) {
+            UpdateTaskStatusCommand newUpdateCommand = (UpdateTaskStatusCommand) inputCommand;
+            updateTaskStatusInList(newUpdateCommand);
+        } else if (inputCommand.getType() == Command.CommandType.PRINTLIST) {
+            printList();
+        }
+    }
+
+    public void printList() {
+        System.out.println("\t Here are the tasks in your list:");
+        for (int i = 0; i < numOfTaskInList; i++) {
+            int taskNumber = i + 1;
```

Comment:
Perhaps you could consider initializing taskNumber outside of the for loop so that you don't have to redeclare it during every iteration of the for loop

Reply:
Agreed. I would bring out the variable taskNumber to prevent redeclaring for every loop instance.